CLI/controllers/update_test.go

```diff
@@ -1,225 +1,204 @@
 package controllers_test
 
 import (
-	"cli/controllers"
 	"cli/models"
 	test_utils "cli/test"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestUpdateTagColor(t *testing.T) {
+// region sizeU
+
+// Test an update of a device's sizeU with heightUnit == mm
+func TestUpdateDeviceSizeUmm(t *testing.T) {
 	controller, mockAPI, _, _ := test_utils.NewControllerWithMocks(t)
 
-	oldSlug := "slug"
-	path := models.TagsPath + oldSlug
+	path := models.PhysicalIDToPath("BASIC.A.R1.A01.chU1")
 
-	test_utils.MockGetObjectByEntity(mockAPI, "tags", map[string]any{
-		"slug":        oldSlug,
-		"description": "description",
-		"color":       "aaaaaa",
-	})
+	device := test_utils.GetEntity("device", "chU1", "BASIC.A.R1.A01", "test")
+	test_utils.MockGetObject(mockAPI, device)
 
 	dataUpdate := map[string]any{
-		"color": "aaaaab",
+		"attributes": map[string]any{
+			"sizeU": 1,
+		},
+	}
+	mockDataUpdate := map[string]any{
+		"attributes": map[string]any{
+			"sizeU":  float64(1),
+			"height": 44.45,
+		},
 	}
 	dataUpdated := map[string]any{
```

Comment:
Shouldn't this be the same as "device" but only with the PATCH values modified ?

Reply:
I didn't think we could reuse the same map, but it works!